Assets/MRTK/Core/Utilities/Gltf/Schema/GltfMeshPrimitive.cs

```diff
@@ -12,10 +12,8 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Gltf.Schema
     /// https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/mesh.primitive.schema.json
     /// </summary>
     [Serializable]
-    public class GltfMeshPrimitive : GltfProperty
+    public class GltfMeshPrimitive : GltfProperty, ISerializationCallbackReceiver
     {
-        #region Serialized Fields
```

Comment:
Why have you removed the Region?  These are there to help organize the code, please revert.